### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by fallenlucario** [gocario/PHBank#10]
Hi, I've been testing this out and it's work wonderfully so far. I'd like to know if it'd be possible to add a TSV and Pokemon SV to the displayed information for each pokemon. That way we'd be able to see the shiny value of each pokemon and the shiny value of it's trainer (if the pokemon is not an egg)

I feel like this would be a useful feature to use and could bring back shiny hatching in the community.

**Comment by gocario** [comment]
What would be really relevant to show? The TID/SID/PID, the PSV/TSV, or..?
Tell me exactly what shall be displayed and how to print them (as hexadecimal, decimal, truncated?).

**Comment by fallenlucario** [comment]
The most relevant thing to show in regards to this would be the PSV (Especially in egg) and TSV (of the game's OT)

As for the printing I don't really understand the question, I'm sorry what do you mean by that ?

**Comment by suloku** [comment]
The PSV and TSV values can be shown in hexadecimal numbers or decimal numbers, gocario is asking what is used as a standard in forums when posting your own values (decimal or hexadecimal)

I've found this:https://docs.google.com/spreadsheets/d/18A9_WbcfbPP-gV86WXARX-rylviBk_9RdvKPAbfYyLs/edit?type=view&gid=0&f=true&sortcolid=-1&sortasc=true&rowsperpage=8000#gid=0

It seems all TSV values are in decimal.

My personal opinion is to show all (TID/SID/PID/PSV), or at least TID/SID/PSV so one can tell a friend's SID via a simple trade.

**Comment by fallenlucario** [comment]
Yeah That would be the most  efficient way (TID/SID/PSV) because PID isn't used for anything really, except maybe calculating PSV but just showing the PSV would make more sense.

Also Suloku, I'm assuming decimal just means regular integers? I was confused by that since all the values are just whole numbers.

**Comment by suloku** [comment]
Hexadecimal: base 16 system (0123456789ABCDEF)
Decinal: base 10 system (0123456789)

**Comment by fallenlucario** [comment]
Understood! One question though, as I'm fairly new to github and compiling. All these commits that are done, they are alterations to the code of the program and will affect the application, correct? So even though an official release hasn't been made yet, could I compile the current files here and try it out to see what's different from the last release? If so, how are these things even compiled, I'm curious...

**Comment by fallenlucario** [comment]
Thank You for including the TID/SID/TSV for the "Trainer" of the game. Only thing that is missing is the PSV (Pokemon Shiny Value). I don't mean to be a bother with it but let me suggest the following:

The New GUI on the top screen would be a lot cleaner if you could have it display information like this:

Trainer: "Trainer Name" ("TID"-"SID"-"TSV") 
Dex No. "Pokemon Name Here" (PSV)

The rest of the lines are perfect as they are. The SID isn't really used for anything like i said before the PSV and TSV are. If you need to make space so the numbers dont go out of the box you can delete the TID/SID part.

Another place where you could put the PSV is under the "OT" and to the left of the word "Value"  I'm pretty sure it'd look neater to have something there like "PSV: XXXX"

Thank you very much for the CIA you've put out, the multi select feature and the new GUI on the bottom screen is great!

**Comment by d4mation** [comment]
+1 to PSV.

It would be very useful for checking Eggs.
